docs/topics/cache.txt

```diff
@@ -363,6 +363,16 @@ user ``apache``, make sure the directory ``/var/tmp/django_cache`` exists and
 is readable and writable by the user ``apache``, or that it can be created by
 the user ``apache``.
 
+.. warning:: 
+
+    Filesystem caching can become slow if you cache a large number of files. If 
+    you run into this problem then you might want to consider another caching 
+    mechanism.  
+
+    If you do need to use filesystem caching for a large number of files then 
+    it is recommended that you subclass ``FileBasedCache`` and improve the 
+    culling strategy.
```

Comment:
Maybe :thinking: :
```suggestion
    Filesystem caching may become slow when storing a large number of files. If
    you run into this problem, consider using a different caching mechanism.
    You can also subclass
    :source:`FileBasedCache <django/core/cache/backends/filebased.py>` and
    improve the culling strategy.
```

Reply:
@sheenarbw This comment is not resolved. Can you take a look at the proposed wording? :point_up: Thanks.

Reply:
@sheenarbw Gentle reminder :arrow_up: Thanks :+1: